hooks/ceph_hooks.py

```diff
@@ -107,6 +107,9 @@
 from charmhelpers.contrib.charmsupport import nrpe
 from charmhelpers.contrib.hardening.harden import harden
 
+os.sys.path.insert(1, os.path.join(sys.path[0], "../files/nagios"))
```

Comment:
The UOS team uses this approach when they need to add the charmhelpers folder to the path. You may want to use the same for the `../files/nagios/` folder.

```
_path = os.path.dirname(os.path.realpath(__file__))
_hooks = os.path.abspath(os.path.join(_path, "../hooks"))

def _add_path(path):
    if path not in sys.path:
        sys.path.insert(1, path)

_add_path(_hooks)
```